### PR TITLE
ncurses 6.0-20160220

### PR DESCRIPTION
**Pull request by vszakats**
* fix to point to a versioned source URL
  (to have a stable content/checksum)
* update source to the latest version
* use sha256 checksums
* fix 'pkgdesc'

Ref: https://github.com/Alexpux/MINGW-packages/pull/1077

### CODE REVIEW

**Comment by vszakats** [comment]
This will properly upgrade while switching to a stable (non-volatile) URL.

File: mingw-w64-ncurses/PKGBUILD [hunk 1]
```diff
@@ -2,24 +2,23 @@
 
 _realname=ncurses
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-_base_ver=6.0
-_date_rev=20150627
-pkgver=${_base_ver}.${_date_rev}
-pkgrel=2
-pkgdesc="System V Release 4.0 curses emulation library"
+pkgver=6.0-20160220
```
Comment (Alexpux) [inline mingw-w64-ncurses/PKGBUILD]: Do not break versioning please. Leave as it was - ver.date

File: mingw-w64-ncurses/PKGBUILD [hunk 2]
```diff
@@ -2,24 +2,23 @@
...
+pkgdesc="System V Release 4.0 curses emulation library (mingw-w64)"
 arch=('any')
 depends=("${MINGW_PACKAGE_PREFIX}-libsystre")
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc")
 url="https://www.gnu.org/software/ncurses/"
 license=('MIT')
 options=('staticlibs' 'strip')
-source=("${_realname}-${pkgver}.tar.gz"::http://invisible-island.net/datafiles/current/${_realname}.tar.gz
+# ftp://invisible-island.net/ncurses/current/${_realname}-${pkgver}.tgz
+source=("http://invisible-mirror.net/archives/ncurses/current/${_realname}-${pkgver}.tgz"
```
Comment (Alexpux) [inline mingw-w64-ncurses/PKGBUILD]: Instead pkgver you can use ${_base_ver}-${_date_rev}

File: mingw-w64-ncurses/PKGBUILD (hunk 1 above)
Comment (vszakats) [inline mingw-w64-ncurses/PKGBUILD]: Done it! (I see the `.` now)

**Pull request merged**